ydb/library/yql/providers/yt/lib/res_pull/res_or_pull.cpp

```diff
@@ -111,11 +113,20 @@ TSkiffExecuteResOrPull::TSkiffExecuteResOrPull(TMaybe<ui64> rowLimit, TMaybe<ui6
     : IExecuteResOrPull(rowLimit, byteLimit, columns)
     , HolderFactory(holderFactory)
     , SkiffWriter(*Out.Get(), 0, 4_MB)
+    , Columns(columns)
 {
     Specs.SetUseSkiff(optLLVM);
     Specs.Init(codecCtx, attrs);
+    YQL_ENSURE(Specs.Outputs.size() == 1);
```

Comment:
Why is AlphabeticPermutations a vector if its length is always 1?

Reply:
Nice observation! Will change it.